.storybook/config.js

```diff
@@ -1,11 +1,28 @@
-import { configure, setAddon, addDecorator } from '@storybook/react';
+import {
+  configure,
+  setAddon,
+  addDecorator,
+  getStorybook,
+} from '@storybook/react';
+import createPercyAddon from '@percy-io/percy-storybook';
 import { withOptions } from '@storybook/addon-options';
 import IntlDecorator from './decorators/intl';
-// import infoAddon from '@storybook/addon-info';
+import './main.mod.css';
 
-// setAddon(infoAddon);
+const { percyAddon, serializeStories } = createPercyAddon();
+setAddon(percyAddon);
 
-import './main.mod.css';
+const srcStories = require.context('../src', true, /\.story\.js$/);
+const materialsStories = require.context('../materials', true, /\.story\.js$/);
+const exampleStories = require.context('../examples', true, /\.story\.js$/);
```

Comment:
Just wondering: example stories contain something from a src story and additional example things we do not really care about too much?

Reply:
Those are these stories which serve as an "overview" and where the code itself contains technical pointers:

![image](https://user-images.githubusercontent.com/1765075/49217793-deb4bf00-f3ce-11e8-90fa-dc625395d303.png).

We had them all along, they're not related to the Visual Regression Testing (but there is no way to exclude stories afaik).